handbook/engineering/developer-insights/code-insights/processes.md

```diff
@@ -16,28 +16,79 @@ Attendees are encouraged to add comments on talking points asynchronously before
 
 ## Planning and prioritization
 
-We plan and track our day-to-day work on our [Kanban board](https://github.com/orgs/sourcegraph/projects/118). Our current process is as follows:
+We plan and track our day-to-day work on our [GitHub board](https://github.com/orgs/sourcegraph/projects/200/views/1).
+The project is separated into multiple views: **Current iteration**, **Next iteration** (both grouped by issue status) and **All iterations** (grouped by milestone).
 
-- Incoming tickets (e.g. from other teams) arrive in the _Inbox_ column.
-- Work is scheduled by adding a card to the _Planned_ column. This happens after talking through the next priorities in our [weekly sync](#weekly-sync) or raising something asynchronously in the #code-insights Slack channel.
-  - The _Planned_ column is an _ordered_ column, by priority. Priority is discussed on the team.
-  - Work should not be moved into the _Planned_ column until it is ready for development _and_ estimated and prioritized.
-  - Work that is ready for development but not yet estimated or prioritized sits in the _Ready for Dev_ column.
-  - Anything that needs design input gets the `needs-design` label and goes in the _Needs design_ column.
-- When starting work, engineers pull cards from the _Planned_ column and move it to the _In Progress_ column.
-- There should never be more than a couple of cards in the _In Progress_ column at the same time. If we paused work on something (e.g. because priorities changed), it should be moved back to _Planned_ or _Icebox_ as appropriate.
-- Things we triaged from the _Inbox_, but don't plan to work on, go into the _Icebox_.
+Our current process is as follows:
 
-Note: we intentionally plan so we can avoid merging significant work less than two days before a release (if a release is on the 20th, our last day to merge is the 18th). Exceptions require explicit approval of both the PM and EM.
+- We work in **2-week iterations**. Every iteration as a GitHub milestone, which is created at least one iteration in advance.
 
-## Tracking Issues
+- Incoming tickets (from team members or from other teams) are added to the [GitHub project](https://github.com/orgs/sourcegraph/projects/200/views/1) and will be visible in the "All iterations" view under "No milestone". This represents our **backlog** of _unscheduled_ work.
```

Comment:
Can we make "are added" less passive voice and/or just include what/who adds them? (Is this you/me? Is it a bot? If I was reading this as an "outsider/CE" it should ideally not leave me questioning "wait, do I need to add them?")

Reply:
Haha ideally a GitHub action but in lieu of that we'll all have to do it manually (and correct if we catch a missing one). Working on automation

Reply:
Cool, automation would be awesome, otherwise can we get semi-automation? We used to have a bot that could automatically put issues in a project kanban column – I'd be okay if we just made a "fake" project board (the old kind) and automated the inbox the same way, then you or I could use that to send things to the new project view.

Reply:
Added full automation.